lib/javaHapiValidator.ts

```diff
@@ -34,8 +76,12 @@ export default class JavaHapiValidator extends Stack {
             tracing: Tracing.ACTIVE,
             environment: {
                 FHIR_VERSION: props.fhirVersion,
+                IMPLEMENTATION_GUIDES_BUCKET: igDeployment.deployedBucket.bucketName,
             },
         });
         this.alias = this.hapiValidatorLambda.currentVersion.addAlias(`fhir-service-validator-lambda-${props.stage}`);
+
+        igDeployment.deployedBucket.grantReadWrite(this.hapiValidatorLambda);
```

Comment:
Why does the Lambda need write and delete?

Reply:
It actually doesn't, just finished testing without them. Removed!